FirebaseCore/Sources/FIRLogger.m

```diff
@@ -87,6 +87,7 @@ void FIRLoggerInitialize(void) {
 }
 
 FIRLoggerLevel FIRGetLoggerLevel(void) {
+  FIRLoggerInitialize();
```

Comment:
This was needed in case `loggerLevel` is called very early and the logger is not yet initialized. I practice, this shouldn't happen.